doc/ExpertOptions.md

```diff
@@ -1450,15 +1450,16 @@ A manifest is a JSON document that contains all the inputs/outputs that would ot
 workflow stage. A manifest can be specified in one of two ways: 
 1. A `.json` input file (see [Manifest JSON](#manifest-json)) is the recommended way to provide inputs in the manifest format.  
 `java -jar dxCompiler.jar -inputs mymanifest.json` will produce `mymanifest.dx.json` that can be passed to `dx run -f mymanifest.dx.json`.
-This is the recommended way to provide inputs in manifest format.
-2. A platform `file-xxx` with content described in  [Intermediate manifest file inputs/outputs](#intermediate-manifest-file-inputs/outputs) 
+2. A platform `file-xxx` with content described in  [Intermediate manifest file inputs and outputs](#intermediate-manifest-file-inputs-and-outputs) 
 section can be used to pass manifest output from a stage of one workflow (including the `output` stage) as input to another workflow.  A 
 typical use case for this scenario is when a user wants to pass manifest output file from a stage (including `output` stage) 
 directly to a new workflow. Also, this scenario might be useful when debugging individual stages of a failing workflow.
 
 ## Manifest JSON
 
-When manifest support is enabled, each applet has an `input_manifest___` input field of type `hash`, which means that it 
+When manifest support is enabled, applet/workflow outputs which are passed from one stage to another (or to the final output
```

Comment:
We don't want to inform the user of what the `input_manifest___` is?

Reply:
It's not as simple. It does have this field, as well as a bunch of others, which I could not find the documentation for. APPS-1309 ticket should address that.

Reply:
Gotcha. Thanks!